templates/partials/question.html

```diff
@@ -76,27 +82,4 @@
   {% else %}
     {{ question_answers }}
   {% endif %}
-{% endset %}
-
-{% if wrap_question_in_fieldset %}
```

Comment:
I'm guessing this entire section has now been superseded by the design system, where in the previous version 6, it was still necessary for some reason?

Reply:
Yeah we have a component that takes care of this markup now